Sources/KlaviyoSwift/KlaviyoState.swift

```diff
@@ -278,6 +290,54 @@ struct KlaviyoState: Equatable, Codable {
 
         return pushTokenData != newPushTokenData
     }
+
+    func buildProfileRequest(apiKey: String, anonymousId: String, properties: [String: Any] = [:]) -> KlaviyoAPI.KlaviyoRequest {
```

Comment:
Moved this over from the state management file over to here just so that all the state methods were in this file. Found it easy to read this way but not super married if you had a valid reason for them to be in the state management file.